dataactcore/models/stagingModels.py

```diff
@@ -509,6 +509,12 @@ class AwardProcurement(Base):
     place_of_perform_county_na = Column(Text)
     base_exercised_options_val = Column(Text)
     base_and_all_options_value = Column(Text)
+    cage_code = Column(Text)
+    inherently_government_func = Column(Text)
+    organizational_type = Column(Text)
+    number_of_employees = Column(Text)
+    annual_revenue = Column(Text)
+    total_obligated_amount = Column(Text)
```

Comment:
Does broker need to match the DAIMS schema v1.2 IDD. under their document this field has a Terse 30 Label `total_dollars_obligated`

Others that are different:
```
- inherently_governmental_funct
- materials_supplies_articles_e
- construction_wage_rate_reqs
```

Reply:
It was already not matching to begin with so we left it (it was only added to AwardProcurement, it was already in DetachedAwardProcurement). We also left `gfe_gfp` when we renamed the full label for it (after talking to Kristine) and every single description tag that we have doesn't end in `tag` like the IDD does.

Additionally, the terse labels go up to 30 characters, with a good portion being 27-29 (like all of the ones you listed). Our standards for column names is 26 characters for broker so a ton will never match no matter what we do. The users never see these columns because we map them to something completely different on our end before they're printed to a D1 file.